interceptors/logging/interceptors.go

```diff
@@ -47,40 +50,102 @@ func (c *reporter) PostCall(err error, duration time.Duration) {
 	if err != nil {
 		logger = logger.With("error", fmt.Sprintf("%v", err))
 	}
-	logger.With(c.opts.durationFieldFunc(duration)...).Log(c.opts.levelFunc(code), fmt.Sprintf("finished %s %s call", c.kind, c.typ))
+
+	if !c.opts.shouldLogRequest(interceptors.FullMethod(c.service, c.method), err) {
+		logger.With(c.opts.durationFieldFunc(duration)...).Log(c.opts.levelFunc(code), fmt.Sprintf("finished %s %s call", c.kind, c.typ))
```

Comment:
we could just do it once I think :hugs: 

`logger.With(c.opts.durationFieldFunc(duration)...)`